starknet_py/net/account/account_client.py

```diff
@@ -370,14 +376,15 @@ async def create_account(
         :param private_key: Private Key used for the account
         :param signer: Signer used to create account and sign transaction
         :return: Instance of AccountClient which interacts with created account on given network
+        :param chain: ChainId of the chain used by the client
```

Comment:
We should be doing some verification here if either signer or chain was provided, same as in constructor.